README.md

```diff
@@ -21,7 +22,7 @@ You'll need the following dependencies:
 * pkg-config
 * valac
 
-Run `meson` to configure the build environment and then `ninja` to build
+To build the application locally, clone this repo to your machine. Then, cd to the directory in terminal and run the following commands to configure the build environment and build the application
```

Comment:
Please keep the original sentence because we use it entirely in our repositories, sorry!